#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_deb_cosmetics.dpatch by Daniel Kahn Gillmor <dkg@fifthhorseman.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cosmetic changes applicable only to debian

@DPATCH@
diff -urNad redir-2.2.1~/redir.man redir-2.2.1/redir.man
--- redir-2.2.1~/redir.man	2005-10-22 21:47:44.067287096 -0400
+++ redir-2.2.1/redir.man	2005-10-22 21:48:34.218662928 -0400
@@ -92,7 +92,7 @@
 .B \--transproxy
 On a linux system with transparent proxying enabled, causes redir to
 make connections appear as if they had come from their true origin.
-(see transproxy.txt in the source archive)
+(see /usr/share/doc/redir/transproxy.txt)
 .TP
 .B \--connect
 Redirects connections through an HTTP proxy which supports the CONNECT 
